latch/registry/table.py

```diff
@@ -21,6 +21,7 @@
 import gql
 import graphql.language as l
 import graphql.language.parser as lp
+import pandas as pd
```

Comment:
we should make pandas an optional dep and if people want this functionality they can opt in by doing `pip install latch[pandas]` or smth

Reply:
pandas is pulled in by flytekit anyways, should probably fix that upstream

Reply:
yee